core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -423,35 +406,26 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     consumer.subscribe(pattern1, new TestConsumerReassignmentListener)
     consumer.poll(50)
```

Comment:
Is it intentional that this one still uses the deprecated overload?

Reply:
Not unintentional anyway. The calls that were using a loop could be transformed without any thinking :wink:. Let me take a quick look at the remaining ones and see which ones are easy to replace. This particular call looks totally unnecessary.